packages/ai/src/ONNXSegmentationController.ts

```diff
@@ -928,34 +928,17 @@ export default class ONNXSegmentationController {
    * Assumes the destination canvas is defined by the canvasPosition value
    */
   mapAnnotationPoint(worldPoint, canvasPosition) {
```

Comment:
This is actually the correct scaling here - it was correct originally for unscaled images, but the updated code was off by nearly 10% for the marker position.